app/Platform/Controllers/DeveloperRecentActivityController.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Controllers;
+
+use App\Community\Enums\AwardType;
+use App\Http\Controller;
+use App\Platform\Enums\UnlockMode;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use App\Platform\Models\LeaderboardEntryLegacy;
+use App\Platform\Models\PlayerAchievement;
+use App\Platform\Models\PlayerBadge;
+use App\Platform\Models\System;
+use App\Site\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use Illuminate\Support\Carbon;
+
+class DeveloperRecentActivityController extends Controller
+{
+    public function __invoke(Request $request): View
+    {
+        $targetUsername = $request->route()->parameters['user'];
+        $foundTargetUser = User::where('User', 'like', $targetUsername)->first();
```

Comment:
Is "like" really desirable? http://localhost:64000/user/Jamira%25/dev/feed

Reply:
I was not aware of this behavior; it is not desirable. I've updated this to match your work in #1998, and also updated a similar call on the Completion Progress page.